src/Packagist/WebBundle/Resources/views/package/view_package.html.twig

```diff
@@ -174,6 +174,16 @@
                             {% endif %}
                         </div>
 
+                        <div class="facts col-xs-12 col-sm-6 col-md-12">
+                            {% if version and version.funding is defined and version.funding|length > 0 %}
+                                <p>
+                                    Fund package maintenance!
+                                    {% for fundingOption in version.funding %}
+                                        <br>&raquo; <a rel="nofollow noopener external noindex" href="{{ fundingOption.url }}">{{ fundingOption.type == 'custom' ? fundingOption.url : fundingOption.type }}</a>
```

Comment:
what if we have multiple funding of the same type (for instance multiple github funding configs, for different people) ?

Reply:
@stof that should indeed be displayed differently for the links then. As per the PR description I was hoping maybe someone else would help make better UI for this or I'll have to give it a shot myself. Definitely not leaving it like this.